internal/apiserver/root.go

```diff
@@ -182,6 +185,11 @@ func initConfig() {
 		}
 	}
 	log.Infof("BasicAuth is %v", BasicAuth)
+
+	// look through all the pgouser secrets in the Operator's
+	// namespace and set a generated password for any that currently
+	// have an empty password set
+	setRandomPgouserPasswords()
```

Comment:
This doesn't seem like "config" to me, so I'd like to see this called by `Initialize()` rather than `initConfig()`.

I only see `initConfig()` called once, so I expect ~L134 should behave the same?

Reply:
Yeah, that should work equally as well. I went back and forth on placement. I'll make that update. Thanks.